src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public static void doWork() {
```

Comment:
The method 'doWork' should not be static. Static methods cannot be overridden, which is likely the intended behavior for a method in a subclass of 'Machine'.